examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    ActorBuilder.set_orbit(actor=S2B, position=sentinel2B_eph[0], velocity=[-sentinel2B_eph[1][0], -sentinel2B_eph[1][1],-sentinel2B_eph[1][2]], epoch=today, central_body=earth)
</pre><p>Please  add linebreaks, it is longer than default jupyter cell length , see https://media.discordapp.net/attachments/752863159592812654/1044949712765198386/image.png</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>